Cryptography/Secret-Key Encryption/Secret-Key-Encryption.md

```diff
@@ -50,42 +69,60 @@ In the following, we list some of the commonly used commands related to Docker a
 we are going to use these commands very frequently, we have created aliases for them in the.bashrcfile
 (in our provided SEEDUbuntu 20.04 VM).
 
+- Run the following commands in the terminal.
+
+```
+cd Labsetup
 ```
-$ docker-compose build # Build the container image
-$ docker-compose up # Start the container
-$ docker-compose down # Shut down the container
 
-// Aliases for the Compose commands above
-$ dcbuild # Alias for: docker-compose build
-$ dcup # Alias for: docker-compose up
-$ dcdown # Alias for: docker-compose down
+```
```

Comment:
Please add rest of the commands and then provide the instruction to build and launch the docker image.